ext/opentelemetry-ext-opentracing-shim/tests/test_shim.py

```diff
@@ -406,6 +406,30 @@ def test_log_kv(self):
             # biggest possible loss of precision.
             self.assertAlmostEqual(result, now, places=6)
 
+    def test_log(self):
+        """Test the `log` method on `Span` objects."""
+        with self.shim.start_span("TestSpan") as span:
```

Comment:
You might want an [`assertLogs`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertLogs) here if you care about the deprecation warnings.

Since #198 `start_as_current_span` is the name of the "default" span creation method. There are two ways to interpret this: that we should use `start_as_current_span` everywhere unless we specifically want to avoid modifying the context because that's what users will do, or that we should use `start_span` in tests that don't rely on the context to check that they really don't. I prefer the former, @carlosalberto may prefer the latter.

Reply:
The TracerShim isn't currently implementing a start_as_current_span method as it's only implementing the opentracing methods as far as I understand. Is this something we want to implement?

Reply:
Nope, you're right. I thought this was the OpenTelemetry tracer, not the shim.